core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -317,41 +391,49 @@ int getHandleIndex(int entryIndex) {
     LookUp lookUp(K key) {
         // binary search sorted part of key array to quickly find node to start search at
         // it finds previous-to-key so start with its next
-        int curr = getEntryField(binaryFind(key), OFFSET_NEXT);
-        int cmp;
+        int curr = getEntryField(binaryFind(key), OFFSET.NEXT);
+        int cmp = -1;
```

Comment:
Have you seen any problem or just common sense?

Reply:
Is it still relevant?